bu/Processor.cpp

```diff
@@ -307,12 +307,19 @@ Processor::writeOctantCompressed(const OctantInfo& o, Index& index, IndexIter po
         }
     }
 flush:
-    flushCompressed(table, view, o);
+    try
+    {
+        flushCompressed(table, view, o);
+    }
+    catch (pdal::pdal_error&)
```

Comment:
Why are we swallowing an error and dropping it on the floor here?

Reply:
If this errors, the output files will be missing and so untwine will fail in general.